catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/ToIcebergSortOrder.java

```diff
@@ -22,6 +22,11 @@
 /** Implement gravitino sort order converter to iceberg sort order. */
 public class ToIcebergSortOrder {
 
+  // Private constructor to prevent instantiation
+  private ToIcebergSortOrder() {
```

Comment:
Maybe we can remove these comments. Because we don't add these for other similar classes. We should keep consistent.

Reply:
done